CHANGELOG.md

```diff
@@ -446,7 +446,7 @@ In this release 246 pull requests were closed.
 * [Issue 5763](https://github.com/spyder-ide/spyder/issues/5763) - Make arrow-key selection in multiple-option tab-completion dialogs "roll over" ([PR 5771](https://github.com/spyder-ide/spyder/pull/5771))

```

Comment:
Please revert the changes to this file and put it in the excluded list. The blanks here come from blanks left in the title of PRs and issues, so I don't want to keep fixing them just for the sake of not avoiding blanks.

Reply:
I'm not sure I understand the point, when it takes zero extra effort to have the script remove them along with the others, they still shouldn't be there, they take up extra space in the file and our future commit hook will remove future spaces automatically (or we could just make sure loghub ``.strip()``s issue/PR titles).

Relative to that zero cost and zero downsides, it would require re-writing multiple commits to add it to the blacklist, test that it works, and revert adding the file, for no apparent gain. I'd rather spend that time adding ``.strip()`` to loghub and fixing the problem properly.

Reply:
I really think this is going too far. I agree with preventing to add blanks to code files, but this is not code. So, as you did with the Github template files, please add this one to the blacklist.

Reply:
Agreed this is going great so far, but we should really stick to code files only at the moment. Could you open an issue over loghub to fix the spacing issue? Thanks

Reply:
I still don't see a logically articulated reason for that distinction; either Markdown counts as code, or it counts as documentation, the latter of which we've removed spaces from for years, and regardless, they do not change the output. Its just extra work for no apparent gain to exclude it, but not enough to make arguing over it further worthwhile, so I made the change you requested.

For these purposes, there's little point fixing the issue with loghub if we don't fix it here, since there will still be spurious whitespace left in the file if we don't clean them up. Why should we care about loghub not stripping trailing spaces if we don't actually care about them in the output?